desktop/src/backends/navigator.rs

```diff
@@ -139,20 +139,20 @@ impl NavigatorBackend for ExternalNavigatorBackend {
             return;
         }
 
-        if self.open_url_mode == OpenURLMode::Confirm {
+        if self.open_links == OpenLinks::Ask {
             let message = format!("The SWF file wants to open the website {}", modified_url);
             // TODO: Add a checkbox with a GUI toolkit
-            let confirm = MessageDialog::new()
+            let ask = MessageDialog::new()
                 .set_title("Open website?")
                 .set_level(MessageLevel::Info)
                 .set_description(&message)
                 .set_buttons(MessageButtons::OkCancel)
                 .show();
-            if !confirm {
+            if !ask {
                 tracing::info!("SWF tried to open a website, but the user declined the request");
                 return;
             }
```

Comment:
I don't think this rename makes sense. "Confirm" was meant as in the user confirming it and pressing the okay button to confirm it. So `if !confirm` makes sense as in "if the user didn't confirm it".
`ask` doesn't make sense here as a variable name (in this branch the user gets asked in any case, the condition checks whether the user confirmed to open the URL or not), therefore I wouldn't change it.